lib/rubocop/cop/rails/squished_heredocs.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      #
+      # Checks SQL heredocs to use `.squish`
```

Comment:
I think we should include some detail here on why using `squish` is a good idea.

Since this cop only applies to SQL heredocs, the name should probably reflect that too (`SquishedSQLHeredocs`)?

Reply:
@eugeneius I changed the name of the cop. 

Regarding the more details, isn't it enough to link to the rails-styleguide?

Reply:
I checked some other cops with style guide references, and many had no justification in their documentation either. 👍